.vscode/launch.json

```diff
@@ -1,11 +1,43 @@
 {
   "version": "0.2.0",
   "configurations": [
-      {
-          "name": "Attach to packager",
-          "cwd": "${workspaceFolder}",
-          "type": "reactnative",
-          "request": "attach"
+    {
```

Comment:
should `.vscode` be in gitignore and removed from the repo?

Reply:
I don't think so, hmm I guess the debug configurations should be global to the repo for easy development? Seems to be the story over at js —https://github.com/getsentry/sentry-javascript/blob/master/.vscode/launch.json

Reply:
I see, if it does not change by OS or something, its fine